### ISSUE TITLE
Possible to lower the animation speed?

### ISSUE THREAD

**Issue by zLaddeus** [BlitzKraig/fvtt-CommunityLighting#12]
I am wondering if it is possible to lower the animation speed.
Currently when on the lowest setting, lights like Torches are moving still really fast.
Maybe not much of an issue, more a request.

Cheers!

**Comment by BlitzKraig** [comment]
Hi,

At their lowest speed torches should be quite slow, but I'll take a look. I can add a speed multiplier if needed, with half and quarter values

**Comment by Lathanael** [comment]
I agree with @zLaddeus, the animation speed of many of the effects (core and module added) is quite fast. For lighting up a place it is too fast for me personally. Thus i tend to use only the static lights as they are not as distracting.

So maybe being able to enter either decimal numbers or have a customizable modifier between 0 and 2 to apply to the animation speed would be helpful.

**Comment by Nuku** [comment]
This is now a core foundry option.
